content/blog/2023/03/camel-observation/index.md

```diff
@@ -0,0 +1,201 @@
+io.micrometer.tracing.Tracer tracer = ...; // <1>
+
+// [Metrics]
+MeterRegistry meterRegistry = new SimpleMeterRegistry(); // <2>
+
+// Setting up ObservationRegistry
+ObservationRegistry registry = ObservationRegistry.create(); // <3>
+// Adding metrics support
+registry.observationConfig().observationHandler(new DefaultMeterObservationHandler(meterRegistry)); // <4>
+// Adding basic tracing support
+registry.observationConfig().observationHandler(new DefaultTracingObservationHandler(tracer)); // <5>
+```
+
+- <1> - for tracing a `Tracer` is needed. Check the [docs](https://micrometer.io/docs/tracing) on more information on how to set it up
+- <2> - for metrics a `MeterRegistry` is needed. Check the [docs](https://micrometer.io/docs/concepts) on more information on how to wotk with Micrometer
+- <3> - an `ObservationRegistry` is required to create `Observation`. Check the [docs](https://micrometer.io/docs/observation) on more information on how to set it up
+- <4> - to add capabilities to `ObservationRegistry` one needs to add handlers. Here you have an example of a handler for metrics
+- <5> - Here you have an example of a handler for tracing. This is a basic sample, for moew information check this [part of the docs](https://micrometer.io/docs/tracing#_handler_configuration)
+
+
+## How does Micrometer Observation work with Camel?
+
+Apache Camel comes with an abstraction of a [`Tracer`](https://camel.apache.org/components/3.20.x/others/tracing.html). It automatically creates spans for Camel Routes.
+
+With Micrometer Observation we've modified the concept of a `Tracer` to allow distingushing between high and low cardinality tags. That in turn allowed us to use the `Tracer` API to create `Observations` instead of spans. That allowed us to automatically create both traces and metrics depending on your `ObservationRegistry` setup.
+
+### Setting up Micrometer Observation with Camel
+
+In order to set up Micrometer Observation with Camel you need to use the new `camel-observation` component and set up a `MicrometerObservationTracer` and register it in the `CamelContext`.
+
+```java
+// Camel
+CamelContext context = ...;
+MicrometerObservationTracer micrometerObservationTracer = new MicrometerObservationTracer();
+
+// Micrometer Core
+MeterRegistry meterRegistry = new SimpleMeterRegistry();
+
+// Micrometer Tracing
+Tracer tracer = ...; 
+Propagator propagator = ...;
+
+// Micrometer Observation setup
+ObservationRegistry observationRegistry = ObservationRegistry.create();
+observationRegistry.observationConfig().observationHandler(new DefaultMeterObservationHandler(meterRegistry));
+observationRegistry.observationConfig().observationHandler(
+    new ObservationHandler.FirstMatchingCompositeObservationHandler(
+        new PropagatingSenderTracingObservationHandler<>(tracer, propagator), 
+        new PropagatingReceiverTracingObservationHandler<>(tracer, propagator), 
+        new DefaultTracingObservationHandler(tracer)
```

Comment:
Remove the trailing whitespace so that the checks pass

```suggestion
        new PropagatingSenderTracingObservationHandler<>(tracer, propagator),
        new PropagatingReceiverTracingObservationHandler<>(tracer, propagator),
        new DefaultTracingObservationHandler(tracer)
```